dashboard/app/models/course_version.rb

```diff
@@ -151,6 +152,25 @@ def recommended?(locale_code = 'en-us')
     latest_stable_version == content_root
   end
 
+  def self.courses_for_unit_selector(unit_ids, user)
+    CourseOffering.single_unit_course_offerings_containing_units_info(unit_ids, user).concat(CourseVersion.unit_group_course_versions_with_units_info(unit_ids, user)).sort_by {|c| c[:display_name]}
+  end
+
+  def self.unit_group_course_versions_with_units(unit_ids, user)
```

Comment:
(minor) For the future, I think it's helpful to be able to look at a file and quickly locate the main intended entry points.  One way this can be done is by marking helper methods such as this one as private or moving all of the helper methods to a private section.

Reply:
Great feedback. I was wondering how to make things in this file clearer. I will updated

Reply:
I tried to do this and it was complaining so I think I did it wrong. I'm not gonna hold the PR on this right now